### PR TITLE
Misuv 8585

### PR DESCRIPTION
**Pull request by Antony100**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -1136,7 +1136,7 @@ paymentAllocation.tableHead.allocated-date                      = Date
 paymentAllocation.tableHead.amount                              = Amount
 paymentAllocation.taxYear										= Tax year {0} to {1}
 paymentAllocation.creditOnAccount                               = Credit on account
-paymentAllocation.moneyOnAccount                                = Money in your account
+paymentAllocation.moneyOnAccount                                = Remaining amount (added to available credit)
 paymentAllocation.moneyOnAccount.agent                          = Money in your client’s account
```
Comment (Antony100) [inline conf/messages:1140]: This message was not requested to be changed. However it seems like it should as it is the agent counterpart to the message that was requested.

Comment (danielmaltz1) [reply conf/messages:1140]: Yeah I agree, we do display this content "money in your client's account" so worth going back to the BA and asking if it should be replaced

Comment (Antony100) [reply conf/messages:1140]: It has been agreed to make the agent message the same as `moneyOnAccount`. Will make the change now.

File: app/views/partials/paymentAllocations/PaymentAllocationsCreditAmount.scala.html [hunk 2]
```diff
@@ -35,21 +35,11 @@
 
 @getLink = @{
     if (creditsRefundsRepayEnabled) {
-        if(isAgent) {
```
Comment (Antony100) [inline app/views/partials/paymentAllocations/PaymentAllocationsCreditAmount.scala.html:38]: Now that the there's only the `moneyOnAccount` message, the check if the user is an agent is no longer required. The message is the same regardless of user type.

**Review by danielmaltz1 — APPROVED** [review]

**Pull request merged**